tox.ini

```diff
@@ -1,19 +1,17 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py{37,38}-{linux,windows}
+envlist = py{37,38,39,310}
+isolated_build = true
 
 [gh-actions]
 python =
     3.7: py37
     3.8: py38
-
-[gh-actions:env]
-PLATFORM =
-    ubuntu-latest: linux
-    windows-latest: windows
```

Comment:
Are we no longer testing in Windows?

Reply:
These options are redundant unless there's a difference in the tox configuration between operating systems. Since we don't have that here, I thought it was good to remove this to make the tox.ini file simpler.